### DISCUSSION TITLE
Reduced Text Input

### DISCUSSION THREAD

**Discussion by AndersonSMed** [vtex/shoreline#823] [Design RFCs]
# Feature Request

- [x] I've searched for any related issues and avoided creating a duplicate issue.

## Is this feature request related to an existing component? Please describe.

I would like to have a new style variation of the TextInput component that would make it look like the print below.

<img width="922" alt="image" src="https://user-images.githubusercontent.com/15149071/218178950-a826f509-b33b-4571-b626-6bbcfdd0308c.png">

## Do you have a suggestion for a new component?

I would say this style could be triggered by a prop in the TextInput component. Just like [material-ui](https://mui.com/material-ui/react-text-field/) does.

## Why is this suggestion important?

It's important so I can use this component in some new pages regarding the new Catalog pages that are under construction.

**Comment by davicostalf** [comment]
Hi Anderson! Before we implement this solution in the library we need to understand if it will also be valuable to other teams. In the meantime, please ask the designers in the project to contribute to our [Team components](https://www.figma.com/file/xawNtclWAYbeypb4d66VUX/Team-Components?type=design&t=qD6ddvvmm1dF7Ywh-1) library. This is important since it gives other teams visibility on this solution.

I also remembered [this article](https://medium.com/google-design/the-evolution-of-material-designs-text-fields-603688b3fe03) about why Google changed the appearance of their text fields. They share some concerns that may be valid to bring up with your team.

Feel free to reach if you have any questions, proposals or want suggestions 😉
